packages/hackathon/src/data/sheets_client.ts

```diff
@@ -282,6 +282,26 @@ class SheetsClient {
     return this.hacker
   }
 
+  firstNameSort(a: IHackerProps, b: IHackerProps): number {
```

Comment:
fwiw this could use localeCompare https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/localeCompare